openslides/users/static/templates/user/user-list.html

```diff
@@ -1,7 +1,127 @@
-<ul>
-    <li ng-repeat="user in users">
-        <a ng-href="user/{{ user.id }}/">{{ user.get_short_name() }}</a>
-        <a ng-href="user/{{ user.id }}/edit/">{{ gettext('Edit') }}</a>
-    </li>
-</ul>
-<a ng-href="user/new/">{{ gettext('New') }}</a>
+<h1>{{ gettext("Participants") }}</h1>
+
+<div id="submenu">
+    <div class="btn-group">
+        <a class="btn btn-primary btn-sm" ng-href="user/new">
+            <i class="fa fa-user-plus fa-lg"></i>
+            {{ gettext('New') }}</a>
+        <a class="btn btn-primary btn-sm dropdown-toggle" data-toggle="dropdown">
+            <span class="fa fa-caret-down"></span></a>
+        <ul class="dropdown-menu">
+            <li><a ng-href="user/new"><i class="fa fa-user fa-fw"></i> {{ gettext('Single participant') }}</a></li>
+            <li><a ng-href="user/new_multiple"><i class="fa fa-group fa-fw"></i> {{ gettext('Multiple participants') }}</a></li>
+        </ul>
+    </div>
+    <a href="#" class="btn btn-default btn-sm">
+        <i class="fa fa-group fa-lg"></i>
+        <span class="optional-small"> Groups</span>
```

Comment:
You can remove the white-space before "Groups". the linebreak after line 16 is also treated as space.